samples/src/main/java/org/opensearch/client/samples/Bulk.java

```diff
@@ -33,27 +36,27 @@
  */
 public class Bulk {
     private static final Logger LOGGER = LogManager.getLogger(Bulk.class);
+    private static OpenSearchClient client;
```

Comment:
Back to the same [comment](https://github.com/opensearch-project/opensearch-java/pull/690#discussion_r1374704666), spreading code around would make it difficult to copy / paste the self-sufficient snippets, please do not use static fields but just have it in `main`